### PR TITLE
updated add_to_app/books sample to pigeon 1.0

### PR DESCRIPTION
**Pull request by gaaclarke**
For 1.0 updates: https://github.com/flutter/packages/blob/master/packages/pigeon/CHANGELOG.md

### CODE REVIEW

**Comment by xster** [comment]
Will look. Also @bryanoltman FYI

File: add_to_app/books/flutter_module_books/lib/api.dart [hunk 1]
```diff
@@ -1,116 +1,196 @@
-// Autogenerated from Pigeon (v0.1.17), do not edit directly.
+// Autogenerated from Pigeon (v1.0.1), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
-// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import
-// @dart = 2.8
+// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import, unnecessary_parenthesis, prefer_null_aware_operators, omit_local_variable_types, unused_shown_name
```
Comment (xster) [inline add_to_app/books/flutter_module_books/lib/api.dart:3]: We should add a comment for each of these explaining why we're skipping these things

File: add_to_app/books/flutter_module_books/pigeon/schema.dart [hunk 2]
```diff
@@ -11,14 +11,12 @@ class Book {
...
+  Thumbnail thumbnail;
 }
 
-// TODO(gaaclarke): add this back when the https://github.com/flutter/flutter/issues/58896
-// crash is resolved.
-// class Thumbnail {
-//   String url;
-// }
+class Thumbnail {
+  String url;
```
Comment (xster) [inline add_to_app/books/flutter_module_books/pigeon/schema.dart:18]: bonus, if you enable this now, we should fill it with some data in the app :)

**Review by xster — APPROVED** [review]

File: add_to_app/books/flutter_module_books/lib/api.dart (hunk 1 above)
Comment (gaaclarke) [reply add_to_app/books/flutter_module_books/lib/api.dart:3]: filed issue for pigeon: https://github.com/flutter/flutter/issues/89671

File: add_to_app/books/flutter_module_books/pigeon/schema.dart (hunk 2 above)
Comment (gaaclarke) [reply add_to_app/books/flutter_module_books/pigeon/schema.dart:18]: Good point, it doesn't seem high priority.  If I get a spare sec later this week I can try throwing it in.

**Pull request merged**